.github/workflows/ci.yml

```diff
@@ -10,7 +10,16 @@ on:
       - '**'
 
 jobs:
-  validate:
+  checks:
+    uses: Workiva/gha-dart-oss/.github/workflows/checks.yaml@v0.1.11
+    with:
+      additional-checks: |
+        no_entrypoint_imports
+
+  build:
+    uses: Workiva/gha-dart-oss/.github/workflows/build.yaml@v0.1.11
+
+  test:
```

Comment:
since this was renamed, I'll update Rosie before this merges to look for the renamed check

Reply:
Updated! Unfortunately had to add each-sub job